### DISCUSSION TITLE
How to set the default route for Capacitor app?

### DISCUSSION THREAD

**Discussion by Dylan190774** [quasarframework/quasar#14356] [CLI - Capacitor / Cordova modes]
I have a project where I need to have a browser-version and an app-version, which look different.
I created 2 different layouts and pages and configured the routes:

```
const routes = [
  {
    path: "/",
    component: () => import("src/layouts/MainLayout.vue"),

    children: [
      {
        name: "main",
        path: "main",
        component: () => import("pages/MainPage.vue"),
      },
  ,{
    path: "/app",
    component: () => import("src/layouts/AppLayout.vue"),

    children: [
      {
        name: "app_main",
        path: "app_main",
        component: () => import("pages/AppMainPage.vue"),
      },
    ]
    
  }
];

export default routes;
```

In the browser everything works fine, but when I create a Capacitor app, it should display the app version.
How do I configure this?

**Comment by metalsadman** [comment]
`/` is your default route what you can try in your `component` prop to load whichever component base on a condition. You can check for `process.env.MODE` see https://quasar.dev/quasar-cli-vite/handling-process-env#example
